docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <doc-section id="searchbarDocs" name="Searchbar">
+    <div class="bs-example">
+      <searchbar
+        :data="searchData" :template="searchTemplate"
+        template-name="search" />
+    </div>
+    <doc-code language="markup">
+      <searchbar
+        :data="searchData" :template="searchTemplate"
+        template-name="search" />
+    </doc-code>
+    <doc-code language="javascript">
+      <!-- eslint-disable vue/html-indent, vue/no-parsing-error, vue/valid-v-html -->
+      new Vue {
+        components: {
+          searchbar
+        },
+        data() {
+          return {
+            searchData: [
+              {
+                'headings': {
+                  'normal-include': 'Normal include',
+                  'establishing-requirements': 'Establishing Requirements',
+                  'include-segment': 'Include segment',
+                  'dynamic-include': 'Dynamic include',
+                  'boilerplate-include': 'Boilerplate include',
+                  'nested-include': 'Nested include',
+                  'html-include': 'HTML include',
+                  'include-from-another-markbind-site': 'Include from another Markbind site',
+                  'feature-list': 'Feature list',
+                },
+                'title': 'Hello World',
+                'src': 'index.md',
+              },
+              {
+                'headings': {
+                  'popover-initiated-by-trigger-honor-trigger-attribute':
+                  'popover initiated by trigger: honor trigger attribute',
+                  'support-multiple-inclusions-of-a-modal': 'Support multiple inclusions of a modal',
+                  'remove-extra-space-in-links': 'Remove extra space in links',
+                },
+                'title': 'Open Bugs',
+                'src': 'bugs/index.md',
+              },
+              {
+                'headings': {
+                  'feature-list': 'Feature list',
+                },
+                'src': 'sub_site/index.md',
+                'title': '',
+              },
+            ],
+            searchTemplate: '<span v-html="item.title | highlight value"></span>><br />'
+              + '<span v-if="item.keywords" v-html="item.keywords | highlightAndExtract value <br />"></span>'
```

Comment:
- Extra `>` in `</span>>`.
- Standardize `<br />` after span.
- Can we do `"item.keywords | extract value | highlight value"` for code reuse and consistency?

Reply:
No, we shouldn't compromise this PR.